app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
the task:
If the visitor does not have a `vaccine` key, it means that he is not vaccinated. In this case, the method must raise a NotVaccinatedError exception.

so we don't have to check that it's None here. also, the expression `"key" in dictionary` inherently checks for the presence of the key within the dictionary's keys. You don't explicitly need to specify .keys(). In case where we will have task
```suggestion
        if "vaccine" not in visitor:
```